### ISSUE TITLE
Specifying playbook for `verify` step does not resolve relative paths

### ISSUE THREAD

**Issue by nlsdfnbch** [ansible-community/molecule#2692]
# Issue Type

- Bug report

# Molecule and Ansible details

```
ansible --version && molecule --version
ansible 2.9.6
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/nls/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/nls/.virtualenvs/ansible-role-bb-consul/lib/python3.6/site-packages/ansible
  executable location = /home/nls/.virtualenvs/ansible-role-bb-consul/bin/ansible
  python version = 3.6.9 (default, Nov  7 2019, 10:44:02) [GCC 8.3.0]
molecule 3.0.5.dev13+ga99c2c2d
   ansible==2.9.6 python==3.6

```

Molecule installation method (one of):

- source
- from git, via pip..

Ansible installation method (one of):

- pip

Detail any linters or test runners used:
- yamllint, ansible-lint
- ansible

# Desired Behavior
Execute the `verify.yml` playbook in `molecule/shared`.

# Actual Behaviour

tl;dr: `verify` step is skipped, as molecule claims no playbook was configured.

directory structure:
```bash
molecule/
  default/
    molecule.yml
  shared/
    converge.yml
    verify.yml
```

File contents
```bash
cat molecule/default/molecule.yml
---
driver:
  name: docker
lint: |
  set -e
  yamllint .
  ansible-lint .

platforms:
  - name: consul-server
    image: "geerlingguy/docker-ubuntu1804-ansible:latest"
    dockerfile: ../shared/Dockerfile.j2
    command: ${MOLECULE_DOCKER_COMMAND:-""}
    volumes:
      - /sys/fs/cgroup:/sys/fs/cgroup:ro
    privileged: true
    pre_build_image: true

provisioner:
  name: ansible
  env:
    ANSIBLE_PYTHON_INTERPRETER: /usr/bin/python3
  options:
    extra_vars:
      consul_config:
        server: yes
        retry_join: ["127.0.0.1"]
        bootstrap_expect: 1

  playbooks:
    converge: ../shared/converge.yml

verifier:
  name: ansible
  env:
    ANSIBLE_PYTHON_INTERPRETER: /usr/bin/python3
  playbooks:
    verify: ../.shared/verify.yml
...
cat molecule/shared/converge.yml
---
- name: Converge
  hosts: all
  roles:
    - role: ansible-role-bb-consul
...
cat molecule/shared/verify.yml
- name: Verify
  hosts: all
  tasks:
    - debug: {msg: "executed"}
```

---

I initially had the verify playbook listed under the `provisioner` key instead of under `verifier`, like so:
```
provisioner:
  name: ansible
  env:
    ANSIBLE_PYTHON_INTERPRETER: /usr/bin/python3
  options:
    extra_vars:
      consul_config:
        server: yes
        retry_join: ["127.0.0.1"]
        bootstrap_expect: 1

  playbooks:
    converge: ../shared/converge.yml
    verify: ../.shared/verify.yml

verifier:
  name: ansible
  env:
    ANSIBLE_PYTHON_INTERPRETER: /usr/bin/python3
...
```

That, however, yields the same result.

The `converge.yml` is executed as expected, which makes this even weirder.

Here's the log of `molecule --debug test`:

https://gist.github.com/nlsdfnbch/3bab0bff514bda71a9a23703dd816eb6

**Comment by nikParasyr** [comment]
Hello @nlsdfnbch 

In both your configurations I see that your verify path is `../.shared/verify.yml` instead of `../shared/verify.yml` based on your directory structure. Can you doublecheck if that is set correct, just in case.

**Comment by nlsdfnbch** [comment]
Fuuuck.

Thanks. 😄
